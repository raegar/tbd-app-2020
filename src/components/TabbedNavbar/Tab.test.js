import React from 'react';
import {render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom';
import Tab from './Tab';
import { BrowserRouter as Router } from 'react-router-dom';

//cleansup
afterEach(cleanup);

it("Renders Tab", () => {
    const { asFragment } = render(<Router><Tab></Tab></Router>);
    expect(asFragment()).toMatchSnapshot();
})

