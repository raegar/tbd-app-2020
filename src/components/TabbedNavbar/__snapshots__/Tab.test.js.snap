// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Tab 1`] = `
<DocumentFragment>
  <li
    class="tab-list-item tab-list-active"
  />
</DocumentFragment>
`;
